content/en/sensitive_data_scanner/setup/telemetry_data.md

```diff
@@ -0,0 +1,215 @@
+---
+title: Telemetry Data
+disable_toc: false
+further_reading:
+  - link: "/sensitive_data_scanner/scanning_rules/library_rules"
+    tag: "Documentation"
+    text: "Learn more about out-of-the-box library rules"
+  - link: "/sensitive_data_scanner/scanning_rules/custom_rules"
+    tag: "Documentation"
+    text: "Learn more about creating custom rules"
+---
+
+## Overview
+
+Sensitive Data Scanner in the Cloud scans telemetry data, such as your application logs, APM events, RUM events, and events from Event Management. The data that can be scanned and redacted are:
+
+- Logs: All structured and unstructured log content, including log message and attribute values
+- APM: Span attribute values only
+- RUM: Event attribute values only
+- Events: Event attribute values only
+
+You submit logs and events to the Datadog backend, so the data leaves your environment before it gets redacted. However, the logs and events are scanned and redacted in the Datadog backend during processing, so sensitive data is redacted before events are indexed in Datadog.
```

Comment:
Same note as above about getting a little lost in all the moving parts in this sentence. Probably makes more sense if you have more of a mental model than I do about what happens in the backend vs. the rest of Datadog, but maybe it's a safe assumption that at least some users will lack that mental model too.

Reply:
Made the same edits as above.

```suggestion
You submit logs and events to the Datadog backend, so the data leaves your environment before it gets redacted. The logs and events are scanned and redacted in the Datadog backend during processing, so sensitive data is redacted before events are indexed and shown in the Datadog UI.
```

Reply:
Love it!